common/js_handle.go

```diff
@@ -191,10 +173,19 @@ func (h *BaseJSHandle) JSONValue() string { // TODO: return error
 		if result, _, err = action.Do(cdp.WithExecutor(h.ctx, h.session)); err != nil {
 			k6ext.Panic(h.ctx, "getting properties for JS handle: %w", err)
 		}
```

Comment:
Why do we panic here? Can we return an error?

Reply:
Good spot, that was missed, changed in 8815b4198a6883c06f3147e389f18d2316a61bab